### PR TITLE
docs: Add labels documentation

### LINKED ISSUE #192: Document `instancetype.kubevirt.io` labels used to decorate objects
**What happened**:

$subject, something along the lines of https://github.com/kubevirt/common-templates/blob/master/templates/LABELS.md

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
Steps to reproduce the behavior.

**Additional context**:

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

Adds missing labels documentation.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #192

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Labels.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+n1.medium    39s
+o1.xlarge    39s
+u1.xlarge    39s
+```
+
+## Generic
+
+### `instancetype.kubevirt.io/version`
+
+The version of common-instancetypes the resources has been generated from.
```
Comment (lyarwood) [inline Labels.md:23]: This isn't correct, that's the version of the instance type class.

I'm also missing common-instancetypes-version.

**Review by 0xFelix — APPROVED** [review]
/approve
/lgtm

**Pull request merged**